Chapter-1-initial-architecture/Src/Fitnet.ArchitectureTests/Conventions/InterfacesConventionsTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.ArchitectureTests.Conventions;
+
+using Common;
+
+public sealed class InterfacesConventionsTests
+{
+    [Fact]
+    internal void ShouldStartWithI()
```

Comment:
We could change convetion in all architecture tests to use Should_Start_With_I. 
@meaboutsoftware  what you think?

Reply:
It will require another package to create in building blocks (so it will complicate our repo) - I would prefer to stay as simple as possible. 

In a real project I would add it, no doubts. What I am afraid is that we will overcomplicate our repo and it should be very simple:

- show that there are architecture tests and show how they can be achieved (without sugar syntax, without extension methods)
- show architecture decision log with real-world examples but not over-complicating it

etc. 

Wdyt?